data-es7.js

```diff
@@ -479,56 +438,596 @@ exports.tests = [
+        return typeof SIMD.float32x4.greaterThan === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.greaterThanOrEqual' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.greaterThanOrEqual === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.lessThan' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.lessThan === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.lessThanOrEqual' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.lessThanOrEqual === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.mul' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.mul === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.neg' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.neg === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.not' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.not === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.notEqual' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.notEqual === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.or' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.or === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.select' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.select === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.shuffle' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.shuffle === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.sub' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.sub === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.swizzle' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.swizzle === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.xor' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.xor === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    }
+  }
+},
+{
+  name: 'Class and Property Decorators',
```

Comment:
Just as a guideline, the capitalisation typically used by the current tests is for all-lowercase, except for built-in classes' names like `Map` or `Object`.